pytest/conftest.py

```diff
@@ -0,0 +1,90 @@
+import json
+import uuid
+
+import pytest
+from django.contrib.auth.models import User
+
+from hs_access_control.models import UserAccess
+from hs_core import hydroshare
+from hs_labels.models import UserLabels
+
+
+@pytest.mark.django_db
+@pytest.fixture(scope="function")
+def resource_with_metadata():
```

Comment:
Fixtures use Inversion-of-Control also known as Dependency Injection https://docs.pytest.org/en/latest/fixture.html